bin/restart_codespaces.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+. ~/nvm/nvm.sh
```

Comment:
This works _for me_ but I'm pretty sure that outside of Codespaces, the location should be `~/.nvm/nvm.sh`

Reply:
For now I think I'll go with a TODO because that USAGE.md needs a _major_ update.